content/applications/websites/livechat/responses.rst

```diff
@@ -48,29 +54,27 @@ entry types an operator can make is displayed.
   - :doc:`/applications/productivity/discuss`
   - :doc:`/applications/productivity/discuss/team_communication`
 
-Helpdesk & Helpdesk search
---------------------------
+Ticket & search_tickets
```

Comment:
To follow the conventions of other headings in this doc, remove the command formatting :)
```suggestion
Ticket & search tickets
```